tests/app/dao/test_users_dao.py

```diff
@@ -42,11 +43,13 @@ def test_create_user(notify_db_session, phone_number):
         'mobile_number': phone_number
     }
     user = User(**data)
-    save_model_user(user)
+    save_model_user(user, password='password', create_user=True)
     assert User.query.count() == 1
-    assert User.query.first().email_address == email
-    assert User.query.first().id == user.id
-    assert User.query.first().mobile_number == phone_number
+    user = User.query.first()
+    assert user.email_address == email
+    assert user.id == user.id
```

Comment:
This line isn't testing anything (it's checking a variable is equal to itself)

Reply:
Good catch, I fixed it now.